### PR TITLE
Add bgnpcgn-aze-Cyrl-Latn-1993 encoded by Manuel

### PR DESCRIPTION
**Pull request by ronaldtse**
For @manuel489

### CODE REVIEW

File: maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+authority_id: bgnpcgn
+id: 1993
+language: azeri
```
Comment (ronaldtse) [inline maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml]: should be `aze` from ISO 639-2

File: maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
+---
+authority_id: bgnpcgn
+id: 1993
+language: azeri
+source_script: Azeri
```
Comment (ronaldtse) [inline maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml]: The source script is `Cyrl` for Cyrillic.

File: maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml [hunk 3]
```diff
@@ -0,0 +1,103 @@
+---
+authority_id: bgnpcgn
+id: 1993
+language: azeri
+source_script: Azeri
+destination_script: Latn
+name: AZERBAIJANI CYRILLIC-ROMAN
```
Comment (ronaldtse) [inline maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml]: Faithful names are better: AZERBAIJANI TABLE OF CORRESPONDENCES CYRILLIC-ROMAN -- BGN/PCGN 1993 Agreement

File: maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+  - An inventory of letter-diacritic combinations, with their Unicode encoding, in addition to the unmodified letters of the basic Roman script is:
+  Ğ (U+011E), ğ (U+011F)
+  Ə (U+018F), ə (U+0259)
+  İ (U+0130), ı (U+0131)
+  Ö (U+00D6), ö (U+00F6)
+  Ü (U+00DC), ü (U+00FC)
+  Ç (U+00C7), ç (U+00E7)
+  Ş (U+015E), ş (U+015F)
+
+  - The Roman-script columns show only lowercase forms but, when applying the table, uppercase and lowercase Roman letters as appropriate should be used.
```
Comment (ronaldtse) [inline maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml:31]: This note means that we need to expand the map for upper and lower cases, which you have done. Thanks 😉

File: maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml (hunk 3 above)
Comment (manuelfuenmayor) [reply maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml]: Correction applied.

File: maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml (hunk 2 above)
Comment (manuelfuenmayor) [reply maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml]: Correction applied.

File: maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml (hunk 1 above)
Comment (manuelfuenmayor) [reply maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml]: Correction applied.

File: maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml [hunk 5]
```diff
@@ -1,10 +1,10 @@
 ---
 authority_id: bgnpcgn
 id: 1993
-language: azeri
-source_script: Azeri
+language: aze
+source_script: Cyrillic
```
Comment (ronaldtse) [inline maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml]: Should be `Cyrl`

**Comment by ronaldtse** [comment]
@manuel489 also, there is no need to upload the PDF when adding a script. I'm thinking we may want to keep the Specification PDFs in a separate git repo...? Thanks!

File: maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml (hunk 5 above)
Comment (manuelfuenmayor) [reply maps/bgnpcgn-aze-Cyrl-Latn-1993.yaml]: Ahh yes, sorry. Error corrected.

**Comment by ronaldtse** [comment]
Thanks @manuel489 !

**Pull request merged**